kube/app/deployment.yml

```diff
@@ -67,48 +68,63 @@ spec:
               {{ else }}
               value: redis
               {{ end }}
-            - name: TZ
-              value: Europe/London
-            - name: NODE_TLS_REJECT_UNAUTHORIZED
-              value: "0"
-            - name: SERVICE_URL
-              {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
-              value: lmr-{{ .DRONE_SOURCE_BRANCH }}.internal.sas-lmr-branch.homeoffice.gov.uk
-              {{ else if eq .KUBE_NAMESPACE .UAT_ENV }}
-              value: lmr.uat.sas-notprod.homeoffice.gov.uk
-              {{ else if eq .KUBE_NAMESPACE .STG_ENV }}
-              value: lmr.internal.stg.sas-notprod.homeoffice.gov.uk
-              {{ else }}
-              value: {{ .PRODUCTION_URL }}
-              {{ end }}
-            {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
-            - name: DATASERVICE_SERVICE_HOST
-              value: dataservice-{{ .DRONE_SOURCE_BRANCH }}
-            - name: DATASERVICE_SERVICE_PORT_HTTPS
-              value: "10443"
-            {{ end }}
-            {{ if or (eq .KUBE_NAMESPACE .BRANCH_ENV) (eq .KUBE_NAMESPACE .UAT_ENV) }}
-            - name: ALLOW_SKIP
-              value: "true"
-            - name: SKIP_EMAIL
-              value: "sas-hof-test@digital.homeoffice.gov.uk"
-            {{ end }}
-            - name: SESSION_SECRET
-              valueFrom:
-                secretKeyRef:
-                  name: session-secret
-                  key: session-secret
             - name: NOTIFY_KEY
               valueFrom:
                 secretKeyRef:
                   name: notify-key
                   key: notify-key
-            - name: PDF_CONVERTER_URL
-              {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
-              value: https://html-pdf-converter-{{ .DRONE_SOURCE_BRANCH }}:10443/convert
-              {{ else }}
-              value: https://html-pdf-converter:10443/convert
-              {{ end }}
+            - name: REDIS_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: redis
+                  key: pass
+            - name: USE_MOCKS
+              value: "false"
+            - name: AWS_USER
+              valueFrom:
+                secretKeyRef:
+                  name: ses
+                  key: smtp_username
```

Comment:
this can be removed

Reply:
I have removed these secret key references which aren't in the new secret created